bugnote_add_inc.php

```diff
@@ -127,6 +127,34 @@
 		}
 	}
 
+	if( access_has_bug_level( config_get( 'upload_bug_file_threshold' ), $f_bug_id ) ) {
+		$t_file_upload_max_num = max( 1, config_get( 'file_upload_max_num' ) );
+		$t_max_file_size = (int)min( ini_get_number( 'upload_max_filesize' ), ini_get_number( 'post_max_size' ), config_get( 'max_file_size' ) );
+?>
+			<tr>
+				<th class="category">
+					<?php echo lang_get( $t_file_upload_max_num == 1 ? 'upload_file' : 'upload_files' ) ?>
+					<br />
+					<?php echo print_max_filesize( $t_max_file_size ); ?>
```

Comment:
remove echo, print_max_filesize is void and does the echo itself
I am aware you copied the code from bug_file_upload_inc.php